src/command/http-command.ts

```diff
@@ -401,13 +401,21 @@ export class HttpCommand implements CommandInterface<HttpOptions> {
 		result.statusCodeName = resp.statusMessage ?? '';
 		result.httpVersion = resp.httpVersion;
 
-		result.timings = {
+		const timings = {
 			firstByte: resp.timings.phases.firstByte ?? null,
 			dns: resp.timings.phases.dns ?? null,
 			tls: resp.timings.phases.tls ?? null,
 			tcp: resp.timings.phases.tcp ?? null,
 		};
 
+		// Fixes https://github.com/szmarczak/http-timer/issues/35
+		if (isIP(cmdOptions.target) && timings.tcp === 0 && timings.dns !== null) {
+			timings.tcp = timings.dns;
```

Comment:
I think there is no reason to add dns to tcp instead of assigning it, as in case of request by IP tcp will always be 0.